dev/validate_bom__test.py

```diff
@@ -752,8 +830,15 @@ def validate_test_requirements(self, test_name, spec, metric_labels):
                                   'IncompatableConfig', metric_labels)
         return False
 
+    if spec['api'] in self.__outbound_service_configs:
+      # No need to wait on service if we are using the outbound service host.
+      service_name = spec['api']
+      self.__wait_on_service_using_host(service_name)
+      return True
```

Comment:
still feels like this is premature. It skips the checks at 845 and beyond.
Coudl wait_on_service() figure out if it should use #836?